FluentTerminal.App/Views/SettingsPages/ShellProfileSettings.xaml

```diff
@@ -264,6 +264,19 @@
                                     <HyperlinkButton Command="{x:Bind AddKeyboardShortcutCommand}" Content="Add" />
                                 </StackPanel>
                                 <views:KeyBindingsView ShowCommandName="False" ViewModel="{x:Bind KeyBindings}" />
+                                <TextBlock Margin="0,0,0,8" Text="Line ending translation on paste" />
+                                <RadioButton
+                                    Content="Do not modify line endings on paste"
+                                    GroupName="LineEndingStyle"
+                                    IsChecked="{x:Bind LineEndingStyle, Mode=TwoWay, Converter={StaticResource LineEndingStyleToRadioButtonConverter}, ConverterParameter='DoNotModify'}" />
+                                <RadioButton
+                                    Content="Convert line endings to CR+LF on paste"
+                                    GroupName="LineEndingStyle"
+                                    IsChecked="{x:Bind LineEndingStyle, Mode=TwoWay, Converter={StaticResource LineEndingStyleToRadioButtonConverter}, ConverterParameter='ToCRLF'}" />
+                                <RadioButton
+                                    Content="Convert line endings to LF on paste"
```

Comment:
I would personally go with shorter descriptions here, e.g.: "Do not modify", "Convert to CRLF", and "Convert to LF". Although it is personal preference, you seem to have done the same in the preview of the settings.

Reply:
The idea is to keep descriptions short when viewing a config, but descriptive when editing the configuration, to allow people to understand the functionality when they go to edit a profile. Leaving it as is for now.

Reply:
Hmm, that is a good motivation 🤔 Thinking back, I think my main motivation for commenting was that you're repeating "line endings ... on paste" unnecessarily, what about changing it to something like ⬇️?

* * *

Line endings translation on paste
- [ ] Do not modify
- [ ] Convert to CR+LF
- [ ] Convert to LF

* * *

Or in case you want to keep "line endings" for clarity ⬇️:

* * *

Line endings translation on paste
- [ ] Do not modify line endings
- [ ] Convert line endings to CR+LF
- [ ] Convert line endings to LF